src/amplitude-client.js

```diff
@@ -801,6 +806,20 @@ AmplitudeClient.prototype._saveGclid = function _saveGclid(urlParams) {
   return gclidProperties;
 };
 
+/**
+ * Try to fetch Facebook Fbclid from url params.
+ * @private
+ */
+AmplitudeClient.prototype._saveFbclid = function _saveFbclid(urlParams) {
+  var fbclid = utils.getQueryParam('fbclid', urlParams);
+  if (utils.isEmptyString(fbclid)) {
+    return;
+  }
+  var fbclidProperties = { fbclid: fbclid };
```

Comment:
I think our practice is to add the quote? @kelvin-lu @jooohhn ?

Reply:
^ this got updated with @kelsonpw's linting - if the key is a single word (e.g. not `{ 'fb click id': fbclid }`), this will update to without quotes!